src/components/HeaderNavLink/HeaderNavLink.tsx

```diff
@@ -57,7 +57,7 @@ export const HeaderNavLink: FC<ItemType> = ({ text, imgSrc, alt }) => (
 		<Item>
 			<Link to={`/${text}`} data-testid={text}>
 				<img src={imgSrc} alt={alt} />
-				<span>{text}</span>
+				<span>{alt}</span>
```

Comment:
Why alt, alt can be a long description text.

Reply:
True!